vnext/local-cli/runWindows/runWindows.js

```diff
@@ -70,6 +81,9 @@ async function runWindows(config, args, options) {
         verbose,
       );
     } catch (e) {
+      if (!e) {
```

Comment:
Shouldn't we fix this in the build util so that it just properly throws the exception? Parsing the log after the fact when we're the ones running msbuild and likely have output feels kind of hacky.

Reply:
yes, that is a good idea :) thanks!

---
In reply to: [405936721](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405936721) [](ancestors = 405936721)